test/jdk/java/awt/datatransfer/HTMLDataFlavors/ManualHTMLDataFlavorTest.java

```diff
@@ -90,202 +98,64 @@ public void drop(DropTargetDropEvent dtde) {
 
             Transferable t = dtde.getTransferable();
             try {
-                Sysout.println("ALL:");
-                Sysout.println(t.getTransferData(DataFlavor.allHtmlFlavor).toString());
-                Sysout.println("FRAGMENT:");
-                Sysout.println(t.getTransferData(DataFlavor.fragmentHtmlFlavor).toString());
-                Sysout.println("SELECTION:");
-                Sysout.println(t.getTransferData(DataFlavor.selectionHtmlFlavor).toString());
+                ManualHTMLDataFlavorTest.log("ALL:");
+                ManualHTMLDataFlavorTest.log(t.getTransferData(DataFlavor.allHtmlFlavor).toString());
+                t.getTransferData(DataFlavor.allHtmlFlavor).toString();
+                ManualHTMLDataFlavorTest.log("FRAGMENT:");
+                ManualHTMLDataFlavorTest.log(t.getTransferData(DataFlavor.fragmentHtmlFlavor).toString());
+                ManualHTMLDataFlavorTest.log("SELECTION:");
+                ManualHTMLDataFlavorTest.log(t.getTransferData(DataFlavor.selectionHtmlFlavor).toString());
             } catch (UnsupportedFlavorException | IOException e) {
                 e.printStackTrace();
             }
```

Comment:
Should exception make the test fail?

Reply:
The author of the test didn't think so, this doesn't seem related to converting to a main program
so I don't want to touch it.

Reply:
I feel as if it were a common practice in the olden days… It's commonly seen in manual tests.

It's worth questioning whether the test should fail if an exception occurs… Likely it should unless the exception *is* expected.

Yeah, this adds up… However, coming back to it later will require even more time if it ever happens again especially if no issue to track it is submitted. Submitting an issue also requires time.